index.html

```diff
@@ -292,8 +296,9 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
         <dd>
           This attribute MUST return the type of the interface represented by
           this <a>PerformanceEntry</a> object.
-          <p class="note">Example `entryType` values defined by other
-          specifications include: <code>"mark"</code> and
+          <p class="note">All `entryType` values are defined in the <a href=
```

Comment:
Nit: I'd add a "relevant" after "the", so that it'd be "the relevant registry"

Reply:
Done.